imu_sensor_broadcaster/src/imu_sensor_broadcaster_parameters.yaml

```diff
@@ -5,17 +5,19 @@ imu_sensor_broadcaster:
     description: "Defines sensor name used as prefix for its interfaces.
     Interface names are: ``<sensor_name>/orientation.x, ..., <sensor_name>/angular_velocity.x, ...,
     <sensor_name>/linear_acceleration.x.``",
-    validation: {
-      not_empty<>: null
-    }
+    # FIXME: Currently does not work with namespace
+    # validation: {
+    #   not_empty<>: null
+    # }
```

Comment:
Can you explain a bit more in what situation it doesn't work.

Reply:
Yes sure:

I don't know exactly what's going on. However, when this is checked it then gets an error:

```bash
    [ERROR] [1713772814.609746661] [test_namespace.test_imu_sensor_broadcaster]: Exception thrown during init stage with message: Invalid value set during initialization for parameter 'sensor_name': Parameter 'sensor_name' cannot be empty
```

1. The problem is probably that adding a namespace causes the frame_id parameter to be searched for, which is no longer there (because it is ~/frame_id)
2. Something else with ControllerInterface

Reply:
I think you have misinterpreted it. This information is need to get the proper state interfaces within the controller. Please look into the following code

https://github.com/ros-controls/ros2_controllers/blob/9e97c00496c239bf4961dc043a3f461aae9b23b8/imu_sensor_broadcaster/src/imu_sensor_broadcaster.cpp#L48-L49